### PR TITLE
Upgrade @tableland/sdk

### PR DESCRIPTION
**Pull request by joewagner**
Upgrade the sdk with new error hints

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
   },
   "dependencies": {
     "@ensdomains/ensjs": "3.0.0-alpha.57",
-    "@tableland/sdk": "4.0.2",
```
Comment (sanderpick) [inline package.json:71]: should it be exact version to match what was there? ie, no `^` ?

**Comment by awmuncy** [comment]
Some tests are failing, but I think that is to be expected with the new errors from the SDK. Should be a quick update to the test files.

**Comment by joewagner** [comment]
> Some tests are failing, but I think that is to be expected with the new errors from the SDK. Should be a quick update to the test files.

Yep! update incoming...

File: package.json (hunk 1 above)
Comment (awmuncy) [reply package.json:71]: It is important for ENS to be pinned, the others it is not.

**Review by awmuncy — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (awmuncy) [reply package.json:71]: Oh, the reason; ENS is in alpha and _will_ break things sometimes. But the pinned version cooperates.

**Pull request merged**